co.headsetapp.headset.metainfo.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>co.headsetapp.headset</id>
+  <metadata_license>MIT</metadata_license>
+  <name>Headset</name>
```

Comment:
The metainfo/desktop file & the icons should be part of upstream. They are not Flathub specific.

Reply:
I'm trying to create the flatpak from a `.deb` repository that we already have (it's proving quite difficult so far). I can use the `.dekstop` file included in the `.deb` instead of providing one directly in this repo, if that's what you mean. The file is not going to change, and I thought it'll be just better to provide it here, rather than edit it as part of the build process.
For the `metainfo` file, I thought this file was Flathub specific, I've never seen it used anywhere else, at least our project doesn't use it anywhere else.

Reply:
Appstream is a generic standard, it is used by all distributions and modern "app stores" on linux.

Reply:
I've switched to using the desktop file from the `.deb`, for the metainfo. I'm going to leave it here for the moment. Flatpak is the only one using it so far.
If I publish the app on other app stores, I'll move the manifest upstream and just reference it from there.

@bilelmoussaoui Regarding the icons, as mentioned somewhere else. The last version of our `.deb` is not using scalable icons, so this is just temporary. Once we release a new version, the icons will also come from the `.deb`. If you prefer, I can use URLs for the icons.

Reply:
Flatpak is not the only thing using. Literally all mainstream distributions use this file.